packages/core/admin/admin/src/content-manager/pages/EditView/Header/index.js

```diff
@@ -204,6 +206,63 @@ const Header = ({
           />
         </Dialog>
       )}
+      {showPublishConfirmation && (
+        <Dialog
+          onClose={onPublishPromptDismissal}
+          title="Confirmation"
```

Comment:
Can we translate this string please?

Reply:
Does this mean adding an entry to src/translations/en.json?

There were translations for this popup removed in https://github.com/strapi/strapi/pull/14399. Should I reintroduce them?

Reply:
Yes, I think we need to bring all these translations back. In case the title wasn't in there before, we'd need to add it at least to the English translation json file.